tailwind.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  theme: {
+    extend: {}
+  },
+  variants: {},
+  plugins: []
+}
```

Comment:
ditto about not being necessary

![image](https://user-images.githubusercontent.com/49499/62618387-5f3f9e00-b914-11e9-853f-43b545d814df.png)

Reply:
good point. I'll add when I need it.